src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/XHMMTargetLikelihoodsCalculator.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import org.apache.commons.math3.distribution.NormalDistribution;
+import org.apache.commons.math3.random.RandomGenerator;
+import org.broadinstitute.hellbender.tools.exome.Target;
+
+/**
+ * Implements the TargetikelihoodCalculator interface for the original XHMM-based germline model.
```

Comment:
typo: TargetikelihoodCalculator.  might as well do a javadoc link while you're at it.